packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/quanticSearchBoxInput.js

```diff
@@ -179,15 +179,18 @@ export default class QuanticSearchBoxInput extends LightningElement {
     this.dispatchEvent(selectSuggestionEvent);
   }
 
-  handleEnter() {
-    const selectedSuggestion =
-      this.suggestionListElement?.getCurrentSelectedValue();
-    if (this.areSuggestionsOpen && selectedSuggestion) {
-      this.sendSelectSuggestionEvent(selectedSuggestion.rawValue);
-    } else {
-      this.sendSubmitSearchEvent();
+  handleEnter(event) {
+    const isLineBreak = this.textarea && event.shiftKey;
+    if (!isLineBreak) {
```

Comment:
Small question, do we want to be able to trigger the submit if we somehow press `shift+enter` to select a suggestion? My understanding is that this fix would prevent that and nothing would happen in that case?

Reply:
No not really, `shift+enter` should not trigger the selection of a suggestions cause it's different from just clicking `enter`.

`shift+enter` is supposed to solely introduce a line break.

Reply:
all good then!